dbt/adapters/spark/impl.py

```diff
@@ -231,8 +231,20 @@ def get_columns_in_relation(self, relation: Relation) -> List[SparkColumn]:
             # return relation's schema. if columns are empty from cache,
             # use get_columns_in_relation spark macro
             # which would execute 'describe extended tablename' query
-            rows: List[agate.Row] = super().get_columns_in_relation(relation)
-            columns = self.parse_describe_extended(relation, rows)
+            try:
+                rows: List[agate.Row] = super().get_columns_in_relation(relation)
+                columns = self.parse_describe_extended(relation, rows)
+            except dbt.exceptions.RuntimeException as e:
+                # spark would throw error when table doesn't exist, where other
+                # CDW would just return and empty list, normalizing the behavior here
+                errmsg = getattr(e, "msg", "")
+                if (
+                    f"Table or view not found: {relation}" in errmsg or
+                    "NoSuchTableException" in errmsg
+                ):
+                    pass
+                else:
+                    raise e
```

Comment:
This is a meaningful change for end users of `adapter.get_columns_in_relation`. I'm happy to have it in before v1.1.0rc1. Could we add a note about it to the changelog?

Reply:
@jtcohen6 Sounds good! I will add a note when updating the code to fix @ueshin 's comment